deploy/backplane/srep/hypershift/service-cluster/10-srep-service-cluster-project.ClusteRole.yml

```diff
@@ -0,0 +1,97 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-srep-service-cluster-project
+rules:
+- apiGroups:
+  - work.open-cluster-management.io
+  resources:
+  - appliedmanifestworks
+  - operator
+  - work
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - cluster.open-cluster-management.io
+  resources:
+  - clusterclaims
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - operator.open-cluster-management.io
+  resources:
+  - klusterlets
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - policy.open-cluster-management.io
+  resources:
+  - policies
+  - configurationpolicies
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - hypershift.openshift.io
+  resources:
+  - awsendpointservices
+  - hostedclusters
+  - hostedcontrolplanes
+  - nodepools
+  verbs:
+  - patch
```

Comment:
Same concerns with management cluster in that we should be careful with patch.
Note that SC will not have these resources anyway as they're only managed on the MC.
But, recommend for consistency:
```suggestion
```